### ISSUE TITLE
Bandmap callsign select

### ISSUE THREAD

**Issue by DL2KI** [foldynl/QLog#116]
From my point of view there is a problem with the callsign selection in the bandap.

A single click on a callsign in the bandmap triggers the "Callsign Whisperer" function. Another single click on the same callsign transfers the callsign to the "New QSO" window.

A double click on a callsign in the bandmap, transfers the callsign directly into the "New QSO" window.

It happens that when you single-click on a callsign, the green cursor (triangles) does not jump to this callsign, but to another one.

In this case, the green cursor jumps to "9A/DL2HWI/P" and back a few times when i click on "EA4DOS/P".

![Screenshot_20220919_122727](https://user-images.githubusercontent.com/26311402/191011314-8a1355af-4ed4-4ad8-bbd7-df532c6f1b17.jpg)

In this case, the same between "G4AJY" and "J6MAT".

![Screenshot_20220919_122843](https://user-images.githubusercontent.com/26311402/191010901-951b25f9-acfd-4742-8f49-305c9c1e55a3.jpg)

And in this case, the same between "W2MV" and "SP2FAP".

![Screenshot_20220919_122950](https://user-images.githubusercontent.com/26311402/191011103-7af7404e-966d-48ba-a870-d48ea77d92dd.jpg)

I have not yet succeeded in finding a connection or a systematic approach to the effects, since the problem does not always occur. Maybe try it out for yourself.

From my point of view, the "Callsign Whisperer" function in the bandmap could be completely omitted and only triggered when the callsign is dialed from the rig. This is the original idea behind this function. Possibly the selection problem is caused here.

The selection of a callsign in the bandmap should then be triggered either by a double click or by a single click, depending on what you prefer.

**Comment by foldynl** [comment]
It is strange. I saw also your video what you sent me via email. I have tested and getting different behavior. During my testing, there was no difference between single and double click. see attached examples below when I used always a single click.
![bandmap](https://user-images.githubusercontent.com/942769/192007497-b283699a-8b2d-4056-b536-5bb1e4f47c71.gif)
![bandmap2](https://user-images.githubusercontent.com/942769/192007502-ebdff2b7-798b-4ebf-bbb8-119cd29ab83d.gif)
